src/components/Results/Results.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react";
+import NewsCard from "../NewsCard/NewsCard";
+import "./Results.css";
+import Preloader from "../Preloader/Preloader";
+import notFound from "../../assets/not-found_v1.svg";
+
+const Results = ({
+  isLoading,
+  error,
+  articles,
+  showMoreArticles,
+  visibleArticles,
+  onCardLike,
+  onCardDelete,
+  savedArticles,
+  isLoggedIn,
+  handleSignin,
+  handleToggleBookmark,
+}) => {
+  if (isLoading) {
+    return (
+      <div className="results results__loading">
+        <Preloader />
+        <div>Searching for news...</div>
+      </div>
+    );
+  }
+
+  if (error) {
+    return (
+      <div className="results">
+        <img src={notFound} className="results-image"></img>
```

Comment:
Every `img` tag must have a **descriptive** `alt` attribute (the **name** of the image is usually there) to show the **alternative** text if the image is not uploaded, and it’s needed for **screen readers**